examples/fluidsim.dx

```diff
@@ -0,0 +1,151 @@
+'## Fluid sim
+Fluid simulation code based on
+[Real-Time Fluid Dynamics for Games](http://www.intpowertechcorp.com/GDC03.pdf
```

Comment:
Does this URL work for other people? I don't know if it's an Australian issue, but I'm getting ad-ware.

Stam has his own page with freely-accessible versions: https://www.josstam.com/publications

Reply:
That's strange!  Fixed.